tests/integration/test_notebooks.py

```diff
@@ -43,57 +46,69 @@ def test_criteo_notebook(tmpdir):
         gpu_id=0,
         batch_size=100000,
     )
+    bench_results = CriteoBenchFastAI().get_epochs(out.splitlines())
+    bench_results += CriteoBenchFastAI().get_dl_timing(out.splitlines())
+    send_results(db, bench_info, bench_results)
 
 
-def test_criteohugectr_notebook(tmpdir):
-    input_path = os.path.join(DATA_START, "criteo/crit_int_pq")
-    output_path = os.path.join(DATA_START, "criteo/crit_test")
-    os.environ["PARTS_PER_CHUNK"] = "1"
-
-    _run_notebook(
-        tmpdir,
-        os.path.join(dirname(TEST_PATH), "examples", "hugectr", "criteo-hugectr.ipynb"),
-        input_path,
-        output_path,
-        # disable rmm.reinitialize, seems to be causing issues
-        transform=lambda line: line.replace("rmm.reinitialize(", "# rmm.reinitialize("),
-        gpu_id="0,1",
-        batch_size=100000,
-    )
-
-
-def test_optimize_criteo(tmpdir):
-    input_path = os.path.join(DATA_START, "criteo/crit_orig")
-    output_path = os.path.join(DATA_START, "criteo/crit_test_opt")
+# def test_criteohugectr_notebook(tmpdir):
```

Comment:
Did you mean to comment out this test?

Reply:
Yes the hugectr test is commented out. Wanted to integrate it in a seperate PR, but I didnt want to loose the code.